wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Location.kt

```diff
@@ -50,7 +50,7 @@ data class Location(
     }
     append(path)
     if (line != -1) {
-      append(" at $line")
+      append(":$line")
```

Comment:
This should really be
```suggestion
      append(':')
      append(line)
```
and the same below. Otherwise you're creating a nested StringBuilder instance to create a String only to append it to the enclosing StringBuilder.

Reply:
There is an R8 feature request to optimize this case, but it's not enabled yet: https://issuetracker.google.com/issues/113859361

Reply:
I keep forgetting about this. Thank you.